src/sentry/utils/snuba.py

```diff
@@ -226,7 +227,7 @@ def get_project_issues(project_ids, issue_ids=None):
     else:
         hashes = GroupHash.objects.filter(project__in=project_ids)
 
-    hashes = list(hashes)
+    hashes = [h for h in hashes if HASH_RE.match(h.hash)]
```

Comment:
https://github.com/getsentry/sentry/blob/83b4746fc6d3be863aafa82e43b52efcd3b6456d/src/sentry/event_manager.py#L50

Reply:
I think that this could also be `hashes.filter(hash__regex=HASH_RE.pattern)` if we'd rather push the filter into the database.

Reply:
I don't know enough about the implications of that query in Postgres to have an opinion, it doesn't seem like it would save a ton anyway?

Super, super nit: since 90 days haven't passed as of this writing, there are groups that exist that may not have the new hash version written, so `set(h for h in hashes if HASH_RE.match(h.hash) else md5(h))` may be "more correct" but I doubt it'll matter much for comparisons and won't matter by the time we are in production (I think) so maybe worth ignoring me?